specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimdiagnostics.json

```diff
@@ -0,0 +1,574 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on PortalSettings entity associated with your Azure API Management deployment. Using this entity you can manage settings for a Developer Portal.",
```

Comment:
>PortalSettings [](start = 69, length = 14)

Not PortalSettings, but enable Diagnostics. What does the entity actually represent?